pkg/local_object_storage/blobstor/exists.go

```diff
@@ -55,8 +74,36 @@ func (b *BlobStor) existsBig(addr *addressSDK.Address) (bool, error) {
 	return err == nil, err
 }
 
-// checks if object is presented in blobovnicza.
-func (b *BlobStor) existsSmall(_ *addressSDK.Address) (bool, error) {
-	// TODO: #1143 implement
-	return false, nil
+// existsSmall checks if object is presented in blobovnicza.
+func (b *BlobStor) existsSmall(addr *addressSDK.Address) (bool, error) {
+	return b.blobovniczas.existsSmall(addr)
+}
+
+func (b *blobovniczas) existsSmall(addr *addressSDK.Address) (bool, error) {
+	activeCache := make(map[string]struct{})
+
+	prm := new(blobovnicza.GetPrm)
+	prm.SetAddress(addr)
+
+	var found bool
+	err := b.iterateSortedLeaves(addr, func(p string) (bool, error) {
+		dirPath := filepath.Dir(p)
+
+		_, ok := activeCache[dirPath]
+
+		_, err := b.getObjectFromLevel(prm, p, !ok)
+		if err != nil {
+			if !errors.Is(err, object.ErrNotFound) {
+				b.log.Debug("could not get object from level",
```

Comment:
why `debug`?

Reply:
It is logged in `Exists` if it needs to be reported.